static/i18n/en/app.json

```diff
@@ -1301,7 +1301,8 @@
       },
       "updatable": {
         "title": "Updates available",
-        "progressTitle": "{{number}} App updates",
+        "progressTitle": "{{number}} App update",
+        "progressTitle_plural": "{{number}} App updates",
```

Comment:
Just to confirm that this isn't actually implemented anywhere in the codebase yet. Is that expected?

Reply:
This is the text shown on the Manager when you have updatable applications on the connected device, yo should see a box listing the applications that could be updated. If there's only one, the text should read "1 App update" and if there are more, "X App updates", same from french which was the cause of the original bug

Reply:
Unfortuantely I'm not seeing that. It's still saying plural for one update 
<img width="633" alt="Screenshot 2021-08-09 at 15 49 47" src="https://user-images.githubusercontent.com/87659842/128717634-6cd04ecc-b3ea-4471-b399-d060f5e57a02.png">

Reply:
You are right, but as @gre said we shouldn't test it in french but rather in English.
I'll change this one but for the rest I'll rely on crowdin

Reply:
You are right, I was missing part of the translations, I updated the pr